GoogleBiddingAdapter/src/main/java/com/chartboost/mediation/googlebiddingadapter/GoogleBiddingAdapter.kt

```diff
@@ -65,6 +67,24 @@ class GoogleBiddingAdapter : PartnerAdapter {
                     )
                 }
             }
+
+        /**
+         * Convert a given Google Bidding error code into a [ChartboostMediationError].
+         *
+         * @param error The Google Bidding error code as an [Int].
+         *
+         * @return The corresponding [ChartboostMediationError].
+         */
+        internal fun getChartboostMediationError(error: Int) =
```

Comment:
not private? where else do we use this?

Reply:
the callback implementations are referencing this, and they're in their own separate classes now.